roles/readlater/tasks/wallabag.yml

```diff
@@ -61,21 +64,20 @@
         recurse=yes
         state=directory
 
-# the httpd only needs write access to the wallabag assets, cache and db directories
-- name: Set wallabag assets, cache and db permissions
+- name: Set wallabag permissions
   file: path=/var/www/wallabag/{{ item }}
         mode=0775
         state=directory
+        owner=www-data
+        group=www-data
+        recurse=true
   with_items:
-    - assets
-    - cache
-    - db
-
-- name: Create the configuration file
-  template: src=var_www_wallabag_inc_poche_config.inc.php.j2
-            dest=/var/www/wallabag/inc/poche/config.inc.php
-            owner=root
-            group=www-data
+    - bin
+    - app/config
+    - vendor
+    - data
+    - app
+    - var
 
 - name: Rename existing Apache wallabag virtualhost
   command: mv /etc/apache2/sites-available/wallabag /etc/apache2/sites-available/wallabag.conf removes=/etc/apache2/sites-available/wallabag
```

Comment:
That's a leftover from the prior implementation, I'll check if it save to remove...

Thanks for your explanation on the upgrade procedure of sovereign! Good to know that backwards compatibility is not necessary.